libs/prov/prov_k8s_cortx_upgrade.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+"""
+Provisioner utility methods for Upgrade of k8s based Cortx Deployment
+"""
+import logging
+import os
+from string import Template
+from commons import commands as common_cmd
+from commons.helpers.pods_helper import LogicalNode
+from libs.prov.prov_k8s_cortx_deploy import ProvDeployK8sCortxLib
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ProvUpgradeK8sCortxLib:
+    """
+    This class contains utility methods for all the operations related
+    to k8s based Cortx Deployment .
```

Comment:
Need to update from `Deployment` to `Upgrade`

Reply:
done